frontend/src/css/Generic.styl

```diff
@@ -11,7 +11,7 @@ a
   vertical-align middle
 
 .outlineTextBtn
-  padding 10px 15px
+  padding 10px 14px
```

Comment:
please don't edit this generic class
I prefer not to because we are on the verge of new version, it's not the right time to change such a generic css rule

Reply:
prefer reducing the font size for now